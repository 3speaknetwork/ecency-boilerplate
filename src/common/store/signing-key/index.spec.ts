import reducer, {initialState, setAct} from "./index";

let state = initialState;

it("1- default state", () => {
    expect(state).toMatchSnapshot();
});

it("2- Set Act", () => {
    state = reducer(state, setAct("swcdwedk23e21dwedewfckoefp"));
    expect(state).toMatchSnapshot();
});
