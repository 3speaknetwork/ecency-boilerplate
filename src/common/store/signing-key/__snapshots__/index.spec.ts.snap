// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1- default state 1`] = `""`;

exports[`2- Set Act 1`] = `"swcdwedk23e21dwedewfckoefp"`;
